src/datamodel/validator/UMessage.cpp

```diff
@@ -155,7 +158,7 @@ ValidationResult isValidRpcRequest(const v1::UMessage& umessage) {
 		return {false, Reason::DISALLOWED_FIELD_SET};
 	}
 
-	return {true, std::nullopt};
+	return {true, {}};
```

Comment:
Soooo, no objection to the change, but I've submitted other code that used std::nullopt instead of {}.  Its a style/preference thing ... I wonder if we could get the linter to "fix" it one way or the other (one-day not this-day)?

Reply:
When default constructing an object, the `{}` style is preferred. In the case of `std::optional`, the default state is empty.

It would be great to teach the linter how to check for these things, although I'm not sure where to start on that.